packages/synapse-react-client/src/components/AccessTokenPage/AccessTokenCard/AccessTokenCard.tsx

```diff
@@ -1,136 +1,173 @@
 import dayjs from 'dayjs'
-import React, { useState } from 'react'
-import { useErrorHandler } from 'react-error-boundary'
-import SynapseClient from '../../../synapse-client'
-import { useSynapseContext } from '../../../utils/context/SynapseContext'
-import { AccessTokenRecord } from '@sage-bionetworks/synapse-types'
-import { scopeDescriptions } from '@sage-bionetworks/synapse-types'
-import { Button, Tooltip } from '@mui/material'
+import React, { useCallback, useState } from 'react'
+import {
+  AccessTokenRecord,
+  scopeDescriptions,
+} from '@sage-bionetworks/synapse-types'
+import {
+  Box,
+  Card,
+  IconButton,
+  Stack,
+  Tooltip,
+  Typography,
+} from '@mui/material'
 import DeleteTwoToneIcon from '@mui/icons-material/DeleteTwoTone'
 import IconSvg from '../../IconSvg/IconSvg'
 import WarningDialog from '../../SynapseForm/WarningDialog'
 import relativeTime from 'dayjs/plugin/relativeTime'
+import { useDeletePersonalAccessToken } from '../../../synapse-queries/user/usePersonalAccessToken'
+import { noop } from 'lodash-es'
 
 dayjs.extend(relativeTime)
 
 export type AccessTokenCardProps = {
   /** Record referring to an access token, not a token itself */
   accessToken: AccessTokenRecord
-  onDelete: (...args: any[]) => void
+  onDelete?: () => void
 }
 
-export const AccessTokenCard: React.FunctionComponent<AccessTokenCardProps> = ({
-  accessToken,
-  onDelete,
-}: AccessTokenCardProps) => {
-  const { accessToken: authToken } = useSynapseContext()
+export const EXPIRED_PAT_WARNING =
+  'This token has expired. It no longer works and can only be deleted.'
+
+export function AccessTokenCard(props: AccessTokenCardProps) {
+  const { accessToken, onDelete = noop } = props
   const [showModal, setShowModal] = useState(false)
-  const handleError = useErrorHandler()
 
   const isExpired = accessToken.state === 'EXPIRED'
 
-  return (
-    <div
-      className={
-        'cardContainer PersonalAccessTokenCard' +
-        (isExpired ? ' bg-warning' : '')
+  const { mutate: deleteToken, isLoading } = useDeletePersonalAccessToken({
+    onSuccess: () => {
+      onDelete()
+    },
+    useErrorBoundary: true,
+  })
+
+  const onClickDeleteButton = useCallback(() => {
+    if (isExpired) {
+      // token no longer works, no need for warning/confirmation
+      deleteToken(accessToken.id)
+    } else {
+      setShowModal(true)
+    }
+  }, [accessToken.id, deleteToken, isExpired])
+
+  const warningDialog = (
```

Comment:
Pulled this dialog JSX out, replaced `p` with `Typography`, simplified `onConfirm`